docs/CHANGELOG.md

```diff
@@ -1,3 +1,169 @@
+  * **pois**: Hide early `cemetery`  (Issue [#1611](https://github.com/tilezen/vector-datasource/issues/1611))
+  * **pois**: Limit zoom range of `range` to zoom 11+.
+  * **pois**: Limit zoom range of `quarry` to zoom 13+ (Issue [#1799](https://github.com/tilezen/vector-datasource/issues/1799))
+  * **pois**: Limit zoom range of `marsh` to zoom 15+ (Issue [#1800](https://github.com/tilezen/vector-datasource/issues/1800))
+  * **pois**: Limit zoom range of `substations` (varies by area) (Issue [#1612](https://github.com/tilezen/vector-datasource/issues/1612))
+  * **pois**: This "park" in Marin shouldn't be `national_park` (instead `common`) (Issue [#1082](https://github.com/tilezen/vector-datasource/issues/1082))
+  * **pois**: Henry Coe State Park is incorrectly marked as kind `national_park` (should be `park`) (Issue [#1728](https://github.com/tilezen/vector-datasource/issues/1728))
+  * **pois**: POIs for `park` labels shown too soon (Issue [#1767](https://github.com/tilezen/vector-datasource/issues/1767))
+  * **pois**: POI labels for `park` show up too late / too early (Issue [#1081](https://github.com/tilezen/vector-datasource/issues/1081))
+  * **pois**: Add `public_transport=station` with `railway=halt` as a synonym for `station` (Issue [#1747](https://github.com/tilezen/vector-datasource/issues/1747))
+  * **pois**: Require names on `station` features.
+  * **pois**: Consolidate `graveyard` and `cemetery` to share same zoom ranges (Issue [#1780](https://github.com/tilezen/vector-datasource/issues/1780))
+  * **pois**: Fix junk `healthcare` values and kinds introduced in v1.5. Added: `chiropractor`, `hospice`, `occupational_therapist`, `optometrist`, `paediatrics`, `physiotherapist`, `podiatrist`, `psychotherapist`, `rehabilitation`, and `speech_therapist` kinds. (Issue [#1596](https://github.com/tilezen/vector-datasource/issues/1596))
+  * **pois**: Use same tier 2 zoom range for `nature_reserve` POIs as the landuse polygons.
+  * **pois**: Fix tier 3 (and 1 & 4) to not duplicate zoom 12 threshold.
+  * **pois**: Fix tier3 POIs zoom 12 threshold to 200000 area (Issue [#1769](https://github.com/tilezen/vector-datasource/issues/1769))
+  * **pois**: Simplify most tier 1 & 2 POI min zooms to area-based thresholds.
+  * **pois**: Default `fitness_station` to zoom 18 instead of 17.
+  * **pois**: Whitelist `kind_detail` for `religion` related kinds to just: `animist`, `bahai`, `buddhist`, `caodaism`, `catholic`, `christian`, `confucian`, `hindu`, `jain`, `jewish`, `multifaith`, `muslim`, `pagan`, `pastafarian`, `scientologist`, `shinto`, `sikh`, `spiritualist`, `taoist`, `tenrikyo`, `unitarian_universalist`, `voodoo`, `yazidi`, and `zoroastrian`.
+  * **pois**: Whitelist `kind_detail` for `health_facility` kind to just types: `CSCom`, `chemist_dispensing`, `clinic`, `counselling_centre`, `dispensary`, `first_aid`, `health_center`, `health_centre`, `hospital`, `laboratory`, `medical_clinic`, `office`, `pharmacy`.
```

Comment:
Merge with the lines describing whitelist changes for **landuse** `kind_detail` in (`religion`, `health_facility`), since the majority of the text is the same?

Reply:
These are different enough to repeat (2x content), plus in Changelog we generally repeat between layers. But reworked in a76eda8099e66fb27aaeea38709ef58b4ec3d6ef.